web/models.py

```
@@ -321,10 +322,8 @@ def available_spots(self):
 
     @property
     def average_rating(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `average_rating`

(ANN201)